### ISSUE TITLE
Node 10.3.0

### ISSUE THREAD

**Issue by BobFrankston** [TimelordUK/node-sqlserver-v8#78]
When I try the Javascript sample I get

```
        throw new Error('None of the binaries loaded successfully. Is your node version either latest 4, 5, 6, 7, 8 or 9 ?')
```

The bin directory doesn't contain a v10 file and copying the v9 as v10 didn't work.

**Comment by TimelordUK** [comment]
It doesn’t seem like you have the latest version checked out. Version 0.6.6 has version 10 binaries in bin folder.

**Comment by TimelordUK** [comment]
i believe this is sorted - I have just published 0.6.7 with binaries for latest v. 10 node
